### ISSUE TITLE
timed modes

### ISSUE THREAD

**Issue by dmodoomsirius** [CloudBotIRC/CloudBot#73]
is it possible to have the admin_channel.py have the ability to have timed modes.
you provide 30s and they will be banned, quieted, and mute and it will do said action for the specified amount of time then remove it.

**Comment by PangeaCake** [comment]
Well.. it's possible.

**Comment by EzraEn1** [comment]
This will be possible when the team releases the python3.4 branch. You might have to wait for a bit.

**Comment by lukeroge** [comment]
Yeah, it will be easier, and you can pretty much steal the code from
reminder
